spec/views/mailchimp/list.json.jbuilder_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+require 'rails_helper'
+
+RSpec.describe '/mailchimp/list.json.jbuilder', type: :view do
+
+	
+  describe 'supporter with no active recurring donations' do 
+		
+    subject(:json) do
+      view.lookup_context.prefixes = view.lookup_context.prefixes.drop(1)
+      assign(:supporter, create(:supporter, name: "Penelope Schultz"))
```

Comment:
😄  at the test name.

Reply:
She often shows up in the code base :smile: